python-listener/src/stream.py

```diff
@@ -119,5 +119,5 @@ def manage_state_change(self, event: dict):
             print("unrecognized state event", event)
 
     def manage_payment_confirmation_required_state_change(self, payload: Payload):
-        print("TODO complete implementation according to the business use-case")
+        # print("TODO complete implementation according to the business use-case")
```

Comment:
What is the motivation behind this commenting? Can you walk me through the use-case of having a commented print?